rust/datafusion/tests/sql.rs

```diff
@@ -530,17 +530,6 @@ async fn sqrt_f32_vs_f64() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test]
-async fn csv_query_error() -> Result<()> {
-    // sin(utf8) should error
-    let mut ctx = create_ctx()?;
-    register_aggregate_csv(&mut ctx)?;
-    let sql = "SELECT sin(c1) FROM aggregate_test_100";
-    let plan = ctx.create_logical_plan(&sql);
-    assert!(plan.is_err());
```

Comment:
I think we should leave this test in (even if we decide that `sin(utf8)` should not error) and update it to show the expected results as a way of documenting the expected behavior.

Reply:
Ok ill re-add.